tox.ini

```diff
@@ -2,6 +2,11 @@
 isolated_build = True
 envlist = py38, py39
 
+[gh-actions]
+python =
```

Comment:
This all looks fantastic, thank you. What is this particular thing for @asinghgaba ?

Reply:
This tells which tox environment to use for the particular python job when running the workflow on Github actions.

Reply:
Is it specifically used by `tox-gh-actions`?

Reply:
yes exactly!

Reply:
Awesome, thanks for explaining that.